### DISCUSSION TITLE
Which kind of chars is used behing groceries emoji in combobox example ?

### DISCUSSION THREAD

**Discussion by MathRobin** [mantinedev/mantine#5798] [Q&A] [answered]
Hi everybody !

In the combobox example https://mantine.dev/combobox/?e=SelectActive, you have put groceries emojis. Works well on every browser met in my company (we have large panel with ubuntu, mac and windows melted).
I guess it was basic unicode emoji characters like in https://www.unicode.org/emoji/charts/full-emoji-list.html
But when change emojis for my business, we needed the country flags, nothing displayed, just letters (see screenshot attache
![2024-02-21_10-35](https://github.com/mantinedev/mantine/assets/1046585/1eabd6a8-f481-482f-affa-b0b16746c8fe)
d)
Am i true and you use unicode emoji or is it something else ?

**Comment by rtivital** [comment]
Regular emojis, like this 🇪🇺. I've copied them from the internet.

**Reply by MathRobin** [reply]
from a specific site ? flags are working on linux and mac but not on windows. I'm afraid about this discussion I found on microsoft forums https://answers.microsoft.com/en-us/windows/forum/all/flag-emoji/85b163bc-786a-4918-9042-763ccf4b6c05

**Reply by rtivital** [reply]
Not really. I've asked ChatGPT to generate a groceries list with emojies and copied it there. Here I've inserted an emoji from Raycast.
If you override font-family in your application, then make sure that it includes emoji font-family as well in the list.

**Reply by MathRobin** [reply, marked as answer]
It seems that Microsoft has really blocked country flags emoji for political reasons. Seems crazy but nothing can be done from mantine, it's on higher level.
https://stackoverflow.com/questions/54519758/flag-emojis-not-rendering/54663926

Thanks for your help
